source/blog/2016-04-27-baseURL.md

```diff
@@ -0,0 +1,104 @@
+
+Because this will have consequences in how your application is deployed we wanted to make sure we provide enough information to help you migrate off of this now-deprecated pattern. This is the primary change that we intend to land in Ember CLI 2.7 which will become our stable release in approximately 10 weeks.
+
+### What was it?
+
+If you're one of the lucky souls who has managed to avoid encountering issues with this, congratulations! If you've had any issues with it, we're sorry. [Variations on the `baseURL` solution has been our default asset loading behavior for two years](https://github.com/ember-cli/ember-cli/commit/404091401400478ef55088888b8ec383e02a17d6#diff-b785f57fd6a80a44da40bd76e531d8ecR11) and you can probably [blame me for it](https://github.com/ember-cli/ember-cli/issues/172).
+
+### Why are we removing it?
+
+There are a number of reasons why we're moving away from this, but a few things really made it unnecessarily complicated.
+
+#### API Design Bugs
+
+Before today Ember CLI apps could have both `baseURL` and `rootURL`. These two names are far too similar and do things which are far too similar. In short, `baseURL` is the property that we used to identify what value to put in the ` ` tag we included in `index.html`. This was to determine where assets external to the application lived, including `app.js`, `vendor.js`, `app.css`, and anything else that wasn't specified with a fully-qualified path. `rootURL` was intended to identify the path at which the Ember application would be hosted so that our router knew which segments should be ignored. We should have consolidated on a single approach long before now.
+
+#### Implicit Behavior
+
+Not only did we have two properties, if only one of `baseURL` or `rootURL` was set Ember CLI tried to guess what the other should be set to based upon the one we did have. This implicit behavior violated the principle of least surprise for many people. If we had always required both to be specified and provided very clear documentation about what each did we may have averted some of the pain.
+
+#### Integration With Other Applications
+
+For example, not every Ember application is full green field development. If you happened to have legacy user-generated content or other data which had relative links, this didn't play nicely with the ` ` tag. Rewriting this content is an option, but incurs cost and bugs we can more-simply avoid by not including the ` ` tag.
+
+#### SVG
+
+And worst of all, in something that we can't code around, SVG has `` cases that require addressing content fragments which break terribly with the ` ` tag. As long as you didn't use SVG all of the other reasons were manageable, but it turns out that `` doesn't play nicely with the ` ` tag. We like like to play nicely with the web ecosystem and breaking weird things in SVG is the final straw that pushed us over the edge on removing the ` ` tag.
```

Comment:
GitHub pops out the `<base>` tag for security reasons. Check out the committed code, it's still there.